angular/controllers/profileController.js

```diff
@@ -1,6 +1,6 @@
 angular.
 	module("recordmate").
-	controller("profileController", ['$scope', '$location', 'userAuth', 'collection', 
+	controller("profileController", ['$scope', '$location', 'userAuth', 'collection', 'ui.bootstrap',
```

Comment:
@adamlucz90 In order to load Angular Bootstrap the 'ui.bootstrap' module [needs to be injected](https://github.com/angular-ui/bootstrap-bower#adding-dependency-to-your-project) into the profileController. This seems to solve the page freezing issue, too.